### DISCUSSION TITLE
pgd attack usage

### DISCUSSION THREAD

**Discussion by COD1995** [Trusted-AI/adversarial-robustness-toolbox#2322] [Q&A] [answered]
How do I apply a pgd attack if the image is normalized [0,1]

Let's say, a 20 -step PGD $\ell_{\infty}$ attack with attack with $\epsilon=8$.

Do I do? 
```

attack = ProjectedGradientDescent(
    estimator=classifier,
    eps=8/255, 
    eps_step=2/255,  
    max_iter=20,  
    num_random_init=1 
)

```

**Comment by beat-buesser** [comment, marked as answer]
Hi @COD1995 Yes, that's correct. You have to provide all eps-values in the same range as the pixels, in your case normalised by 255.
